### PR TITLE
allow to add password protected turn server

### LINKED ISSUE #34: Using external ICE server stun/turn
Hello,

I don't know if it's the right place to ask the question or if I should ask in the original repo.

I would like to install neko in my home lab and expose it on internet. The issue is that I have only a single public IP and multiple services are currently using ICE/stun/turn. I can't therefore forward those port to Neko.

I've however a coturn installed and it works well with my other apps. The question now is how to tell Neko to use that server and not the one built-in

I've configured the reverse proxy on my web server. - That part works well

I've tried on nurdism build to add in the docker-compose the following
```
- NEKO_ICELITE: 0
- NEKO_ICESERVER: 'stun:<my-stun.tld:port> turn:<my-turn.tld:port:secret>'
- NEKO_IP: <my public ip>
- NEKO_NAT1TO1: <private ip of neko>
```

but I have no clue how to specify the turn server, how should I put the secret, etc...
It works if I connect from another machine from inside, but doesn't works from outside.

I'm now running m1k1o/neko:latest but still don't know how to use an external turn.

Thanks for your help !

### PR DESCRIPTION
**Pull request by mbattista**
fixes #34 

Right now it panics on wrong sematics. Should I change it that it defaults back to the google stun?

### CODE REVIEW

File: client/src/neko/base.ts [hunk 1]
```diff
@@ -202,7 +202,7 @@ export abstract class BaseClient extends EventEmitter<BaseEvents> {
     this._peer = new RTCPeerConnection()
     if (lite !== true) {
       this._peer = new RTCPeerConnection({
-        iceServers: [{ urls: servers }],
+        iceServers: JSON.parse(servers),
```
Comment (m1k1o) [inline client/src/neko/base.ts:205]: I'd say, having it as data structure passing to client would be nicer. Now it is JSON carried in JSON as string.

**Comment by m1k1o** [comment]
I'd say panic is expected, if someone is changing that value.

File: client/src/neko/base.ts (hunk 1 above)
Comment (mbattista) [reply client/src/neko/base.ts]: You are right. Now its cleander

**Comment by m1k1o** [comment]
I would not `ICESERVER` call as deprecated. They are just simple usage, and should be available along with password-protected servers.

**Comment by mbattista** [comment]
> I would not `ICESERVER` call as deprecated. They are just simple usage, and should be available along with password-protected servers.

I called it deprecated because it was overwritten when both values are given.
Your solution is neat and invalidates the deprecation!

But it always adds "stun:stun.l.google.com:19302" to the server list, which is not a big deal, since it gets added to the end.
I have zero problem with it and no intentions of fixing it. But if someone wants to change it, both cobra default values have to be empty and only if both are empty add the google stun server.

**Comment by m1k1o** [comment]
Considering the order of servers, that this one is added at the end of list always, it should be only the last resort, if your custom servers fail. So I think it is accepbtable. It could be overwritten in config file, if someone really wanted. But that is again an undocummented feature of this project.

**Pull request merged**